src/dataLoaders/components/collectorsWizard/verify/PluginConfigSwitcher.tsx

```diff
@@ -0,0 +1,143 @@
+// Libraries
+import React, {FC, useState} from 'react'
+import Loadable from 'react-loadable'
+import {connect, ConnectedProps} from 'react-redux'
+
+// Components
+import {Overlay} from '@influxdata/clockface'
+import {PluginCreateConfigurationFooter} from 'src/writeData/components/PluginCreateConfigurationFooter'
+
+// Types
+import {AppState} from 'src/types'
+// Actions
+import {notify as notifyAction} from 'src/shared/actions/notifications'
+import {
+  clearSteps,
+  decrementCurrentStepIndex,
+  incrementCurrentStepIndex,
+  setCurrentStepIndex,
+  setSubstepIndex,
+} from 'src/dataLoaders/actions/steps'
+import {clearDataLoaders} from 'src/dataLoaders/actions/dataLoaders'
+
+// Constants
+import {BUCKET_OVERLAY_WIDTH} from 'src/buckets/constants'
+
+const PLUGIN_CREATE_CONFIGURATION_OVERLAY_DEFAULT_WIDTH = 1200
+const PLUGIN_CREATE_CONFIGURATION_OVERLAY_OPTIONS_WIDTH = 480
+const spinner = <div />
+
+const PluginCreateConfigurationStepSwitcher = Loadable({
+  loader: () =>
+    import('src/writeData/components/PluginCreateConfigurationStepSwitcher'),
+  loading() {
+    return spinner
+  },
+})
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = ReduxProps
+
+export const PluginConfigSwitcher: FC<Props> = props => {
+  const {
+    currentStepIndex,
+    notify,
+    onClearSteps,
+    onClearDataLoaders,
+    onDecrementCurrentStepIndex,
+    onIncrementCurrentStepIndex,
+    onSetSubstepIndex,
+    substepIndex,
+    telegrafPlugins,
+  } = props
+
+  const handleDismiss = () => {
+    onClearDataLoaders()
+    onClearSteps()
+  }
+
+  const [pluginConfig, setPluginConfig] = useState<string>('')
+  const [isValidConfiguration, setIsValidConfiguration] = useState<boolean>(
+    false
+  )
+
+  const pluginConfigName = telegrafPlugins[0].name
+
+  const stepProps = {
+    currentStepIndex,
+    isValidConfiguration,
+    notify,
+    onDecrementCurrentStepIndex,
+    onExit: handleDismiss,
+    onIncrementCurrentStepIndex,
+    onSetSubstepIndex,
+    pluginConfig,
+    setIsValidConfiguration,
+    setPluginConfig,
+    substepIndex,
+    pluginConfigName,
```

Comment:
if there's no clear organization scheme, alphabetization is always a good one to use

Reply:
good deal